stubs/pynput/pynput/keyboard/_base.pyi

```diff
@@ -25,71 +25,71 @@ class KeyCode:
     def from_dead(cls, char: str, **kwargs: Any) -> Self: ...
 
 class Key(enum.Enum):
-    alt: int
-    alt_l: int
-    alt_r: int
-    alt_gr: int
-    backspace: int
-    caps_lock: int
-    cmd: int
-    cmd_l: int
-    cmd_r: int
-    ctrl: int
-    ctrl_l: int
-    ctrl_r: int
-    delete: int
-    down: int
-    end: int
-    enter: int
-    esc: int
-    f1: int
-    f2: int
-    f3: int
-    f4: int
-    f5: int
-    f6: int
-    f7: int
-    f8: int
-    f9: int
-    f10: int
-    f11: int
-    f12: int
-    f13: int
-    f14: int
-    f15: int
-    f16: int
-    f17: int
-    f18: int
-    f19: int
-    f20: int
+    alt = 0
```

Comment:
It appears these values are actually KeyCode object, not ints. I'll try to push a fix.

Reply:
Oh wait, that wasn't released. In the last release all the values are really 0.

Reply:
For reference: https://github.com/moses-palmer/pynput/commit/4ee72f900d92df1ae5c5dd1388a6553ef3f3eaa6. The last release was in January 2022.